patches/0037-Restore-legacy-bucket.patch

```diff
@@ -5,54 +5,1179 @@ Subject: [PATCH] Restore legacy bucket
 
 
 diff --git a/go.mod b/go.mod
-index 87a0092938..b4265d86f7 100644
+index 0d8ccda18d..ee2ff5cbfc 100644
 --- a/go.mod
 +++ b/go.mod
-@@ -256,6 +256,7 @@ require (
+@@ -1,6 +1,6 @@
+ module github.com/hashicorp/terraform-provider-aws
+ 
+-go 1.23.0
++go 1.22.6
```

Comment:
This change seems a bit odd to me.
Maybe I'm reading it wrong, but aren't we downgrading the go version and a lot of the other dependencies here? Why is that needed for the Bucket patch?

Reply:
It might be an artifact of my local tooling versions that's not needed. I'll have a look.